apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/(sidebar)/rpc/components/MethodsTable.tsx

```diff
@@ -0,0 +1,107 @@
+"use client";
+import { useMemo } from "react";
+import type { ThirdwebClient } from "thirdweb";
+import { shortenLargeNumber } from "thirdweb/utils";
+import { ScrollShadow } from "@/components/ui/ScrollShadow/ScrollShadow";
+import { SkeletonContainer } from "@/components/ui/skeleton";
+import type { RpcMethodStats } from "@/types/analytics";
+import {
+  CardHeading,
+  TableData,
+  TableHeading,
+  TableHeadingRow,
+} from "../../universal-bridge/components/common";
+
+export function TopRPCMethodsTable(props: {
+  data: RpcMethodStats[];
+  client: ThirdwebClient;
+}) {
+  const tableData = useMemo(() => {
```

Comment:
instead of just slicing - show pagination. Use `PaginationButtons`

Reply:
no need to add pagination in endpoint, I'm referring to `.slice()` done here - we're not showing all the data we're loading